src/action/keys/index.js

```diff
@@ -0,0 +1,45 @@
+import { ECPair } from 'bitcoinjs-lib';
+import { NEW_KEYS } from '../../constants/actions';
+import { getHexString } from '../../scripts/utils';
+
+class GenerateKeys {
+  constructor() {
+    const map = new Map();
```

Comment:
Why is a map needed if there is only a single entry in it anyways?

Reply:
Correct,  I forgot to change it I was planning something else initially.